Header header
string strName
string strCommand
